src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs

```diff
@@ -94,5 +113,57 @@ internal class ThreadInfo
             public TimeSpan PrivilegedProcessorTime { get; set; }
             public TimeSpan UserProcessorTime { get; set; }
         }
+
+        public static void OutputResultToStream(StreamWriter sw, HashSet<string> threadIds = null, bool includeStackObjects = false)
+        {
+            var ravenDebugExec = Path.Combine(AppContext.BaseDirectory, "Raven.Debug.exe");
+            if (File.Exists(ravenDebugExec) == false)
+                throw new FileNotFoundException($"Could not find debugger tool at '{ravenDebugExec}'");
+
+            using (var currentProcess = Process.GetCurrentProcess())
+            {
+                var sb = new StringBuilder($"stack-traces --pid {currentProcess.Id}");
+
+                if (threadIds != null && threadIds.Count > 0)
+                {
+                    foreach (var threadId in threadIds)
+                    {
+                        sb.Append($" --tid {threadId}");
```

Comment:
Below you parse the values to ensure they are integers. We should have the same here I think

Reply:
done